src/policy/policy.cpp

```diff
@@ -49,7 +49,9 @@ bool IsStandard(const CScript& scriptPubKey, txnouttype& whichType)
             return false;
         if (m < 1 || m > n)
             return false;
-    }
+    } else if (whichType == TX_NULL_DATA &&
+               (!GetBoolArg("-datacarrier", true) || scriptPubKey.size() > nMaxDatacarrierBytes))
```

Comment:
It is clearer if you read commit by commit:
https://github.com/jtimon/bitcoin/commit/dbc241dfd51623da770725909bb329f2188a5cb1
https://github.com/jtimon/bitcoin/commit/904d59368899c9e522325e1925b500b3f9e34ee9
https://github.com/jtimon/bitcoin/commit/df698329c3fd851f2e8028b6879d8169d7f3a37c

The simplest to maintain the current "effective size" of 80 is to simply add 3 to MAX_OP_RETURN_RELAY.
But the 3 is quite arbitrary: the number of pushes depends on the size of the data. So I think it's simpler to count the op_return and the pushes as part of the data: simply reduce the "effective size" from 80 to 77 and change the tests instead of the constant (as in the last commit).